packages/core/src/components/slider/readme.md

```diff
@@ -29,9 +29,10 @@
 
 ## Events
 
-| Event       | Description                                 | Type                              |
-| ----------- | ------------------------------------------- | --------------------------------- |
-| `tdsChange` | Sends the value of the slider when changed. | `CustomEvent<{ value: string; }>` |
+| Event       | Description                                                                                      | Type                              |
+| ----------- | ------------------------------------------------------------------------------------------------ | --------------------------------- |
+| `tdsChange` | Sends the value of the slider when changed. Fires after mouse up and touch end events.           | `CustomEvent<{ value: string; }>` |
```

Comment:
Since the `tdsChange` used to run in every change (input) this would be a breaking change, no?

Reply:
Indeed it is, but I will do my best to speak to users in the announcements channel about this one. Pretty much those that do not need "realtime" data, can keep using tdsChange. For those that want exactly what tdsChange has done so far, they need to use tdsInput instead.

Reply:
Alright, I have no idea where you are at for 2.0. But if this absolutely can't wait for that then go for it 👍🏻